programs/svm-spoke/src/instructions/bundle.rs

```diff
@@ -106,9 +106,12 @@ impl RelayerRefundLeaf {
     }
 }
 
-pub fn execute_relayer_refund_leaf<'info>(
-    ctx: Context<'_, '_, '_, 'info, ExecuteRelayerRefundLeaf<'info>>,
-) -> Result<()> {
+pub fn execute_relayer_refund_leaf<'c, 'info>(
+    ctx: Context<'_, '_, 'c, 'info, ExecuteRelayerRefundLeaf<'info>>,
+) -> Result<()>
+where
+    'c: 'info,
+{
```

Comment:
Added the constraint that lifetime for remaining accounts (`'c`) is at least as long as `'info` for accounts that are referenced by this account array. I think its needed because of manual remaining account processing in `try_from_remaining_account` method. Without these lifetime constraints the compiler errors.

Reply:
o that's interesting. ye, I've had other lifetime issues while implementing other things and was able to address it through other syntax but this is more complex here.